test/e2e/framework/ginkgowrapper.go

```diff
@@ -186,24 +181,21 @@ func (f *Framework) Describe(text string, args ...interface{}) bool {
 //
 // Text and arguments may be mixed. The final text is a concatenation
 // of the text arguments and special tags from the With functions.
-func Context(text string, args ...interface{}) bool {
-	return registerInSuite(ginkgo.Context, text, args)
+func Context(args ...interface{}) bool {
+	return registerInSuite(ginkgo.Context, args)
 }
 
 // Context is a shorthand for the corresponding package function.
-func (f *Framework) Context(text string, args ...interface{}) bool {
-	return registerInSuite(ginkgo.Context, text, args)
+func (f *Framework) Context(args ...interface{}) bool {
+	return registerInSuite(ginkgo.Context, args)
 }
 
 // registerInSuite is the common implementation of all wrapper functions. It
 // expects to be called through one intermediate wrapper.
-func registerInSuite(ginkgoCall func(text string, args ...interface{}) bool, text string, args []interface{}) bool {
+func registerInSuite(ginkgoCall func(text string, args ...interface{}) bool, args []interface{}) bool {
 	var ginkgoArgs []interface{}
 	var offset ginkgo.Offset
 	var texts []string
-	if text != "" {
-		texts = append(texts, text)
-	}
```

Comment:
yeah, but you remove this line the text variable is still missing

Reply:
That was the explicit first text parameter in e.g. `Describe`. That's now handled when iterating over args in https://github.com/kubernetes/kubernetes/blob/6f2591866ed8a479743ca5cb54eaf09a3a3139d4/test/e2e/framework/ginkgowrapper.go#L219-L223

But perhaps I misunderstand the concern?

Reply:
`func registerInSuite(ginkgoCall func(text string, args ...interface{}) bool, args []interface{}) bool {`

that text argument seems to be only used here, if you remove the text parameter is not used anywhere else

Reply:
It's used inside `ginkgo.Describe`, which has `string, ...interface{}` as parameters. Let me remove the parameter name, perhaps that avoids the confusion:

https://github.com/kubernetes/kubernetes/compare/6f2591866ed8a479743ca5cb54eaf09a3a3139d4..81b960efefebbc2cbcb8fd6770d0ef7af30be318